mkdocs.yml

```diff
@@ -132,19 +132,15 @@ nav:
       - 'walk-through/Ch-WalkthroughExporting.md'
   - API Reference:
     - Introduction: './api/Ch-APIIntroduction.md'
-    - Core:
-      - './api/core/Ch-APICoreCommand.md'
-      - './api/core/Ch-APICoreData.md'
-      - './api/core/Ch-APICoreMetadata.md'
-      - './api/core/Ch-APICoreConfigurationAndRegistry.md'
-    - Support:
-      - './api/support/Ch-APISupportNotifications.md'
-      - './api/support/Ch-APISupportScheduler.md'
-    - Device:
-        - './api/devices/Ch-APIDeviceSDK.md'
-    - Application:
-        - './api/applications/Ch-APIAppFunctionsSDK.md'
-        - './api/applications/Ch-APIRulesEngine.md'
+    -  './api/core/Ch-APICoreCommand.md'
+    - './api/core/Ch-APICoreData.md'
+    - './api/core/Ch-APICoreMetadata.md'
+    - './api/core/Ch-APICoreConfigurationAndRegistry.md'
```

Comment:
IMO, this should not be in the middle of the list of services and should be at the bottom or removed.

Reply:
I can reorder to whatever is needed. Please let me know what order you want to see.

Reply:
Ahh, yes we have an issue to get that done. I'll take care of it today.